src/Type/SchemaValidationContext.php

```diff
@@ -521,9 +521,9 @@ private function validateFields($type)
     /**
      * @param Schema|ObjectType|InterfaceType|UnionType|EnumType|InputObjectType|Directive $obj
      *
-     * @return ObjectTypeDefinitionNode[]|ObjectTypeExtensionNode[]|InterfaceTypeDefinitionNode[]|InterfaceTypeExtensionNode[]
+     * @return array<ObjectTypeDefinitionNode>|array<ObjectTypeExtensionNode>|array<InterfaceTypeDefinitionNode>|array<InterfaceTypeExtensionNode>
```

Comment:
Isn't this 

```suggestion
     * @return list<ObjectTypeDefinitionNode|ObjectTypeExtensionNode|InterfaceTypeDefinitionNode|InterfaceTypeExtensionNode>
```

?

Reply:
It was even more complex, but i found and fixed a bug while writing out the type.

Reply:
hah, now it's ultimate 😄